cmd/spiffe-aws-assume-role/cli/cli_test.go

```diff
@@ -17,6 +20,21 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func TestErrorLogging(t *testing.T) {
+	deleteFileIfExists(t, logFileName)
+	defer deleteFileIfExists(t, logFileName)
+
+	badArgument := uuid.New().String()
+	args := []string{badArgument}
+	err := RunWithDefaultContext(args)
+	require.Error(t, err)
+
+	bytes, err := ioutil.ReadFile(logFileName)
+	require.NoError(t, err)
+	logs := string(bytes)
+	require.True(t, strings.Contains(logs, badArgument))
+}
```

Comment:
After ensuring that we start from a clean slate, we pass in a purposely bad command line argument and then verify that it shows up in the logs.